_benchmark/user-guide/optimizing-benchmarks/expand-data-corpus.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Expand data corpus
+nav_order: 20
+parent: Optimizing benchmarks
+grand_parent: User guide
+---
+
+# Expanding the data corpus of a workload
+
+This tutorial shows you how to use the [`expand-data-corpus.py`](https://github.com/opensearch-project/opensearch-benchmark/blob/main/scripts/expand-data-corpus.py) script to increase the size of the data corpus for an OpenSearch Benchmark workload. This is helpful when running the `http_logs` workload against a large OpenSearch cluster.
```

Comment:
```suggestion
This tutorial shows you how to use the [`expand-data-corpus.py`](https://github.com/opensearch-project/opensearch-benchmark/blob/main/scripts/expand-data-corpus.py) script to increase the size of the data corpus for an OpenSearch Benchmark workload. This can be helpful when running the `http_logs` workload against a large OpenSearch cluster.
```